jobs/pgbackup/spec

```diff
@@ -27,22 +27,21 @@ properties:
     default: pgdumpall
   pgbackup.host:
     description: "Hostname or IP address to connect to PostgreSQL on"
-    default: "localhost"
+    default: "127.0.0.1"
   pgbackup.port:
     description: "TCP port to connect to PostgreSQL on"
     default: 5432
   pgbackup.username:
     description: "User to connect to PostgreSQL as"
-    default: "pgbackup"
+    default: "PLEASE-CHANGE-THIS"
```

Comment:
When we use the release to backup a single database , pgbackup.username is used for restore.
If we remove the default and we want to perform a single database backup, we will have to add unused information to our manifest (if we don't want to restore a database using the release)